Packs/GreyNoise/Integrations/GreyNoise/GreyNoise_description.md

```diff
@@ -6,3 +6,6 @@ To use GreyNoise on Cortex XSOAR, retrieve your user account's API key and enter
 
 - [Login](https://viz.greynoise.io/login) / [Register](https://viz.greynoise.io/signup) at GreyNoise
 - Once signed in, go to [Account Settings](https://viz.greynoise.io/account/) to get the API Key.
+
+ 
+---
```

Comment:
@NicCas remove these empty lines, just to remove the diff

Reply:
Completed